clients/src/main/java/org/apache/kafka/clients/admin/ExpireDelegationTokenOptions.java

```diff
@@ -28,6 +28,14 @@
 public class ExpireDelegationTokenOptions extends AbstractOptions<ExpireDelegationTokenOptions> {
     private long expiryTimePeriodMs = -1L;
 
+    /**
+     * When {@link Admin#expireDelegationToken(byte[], ExpireDelegationTokenOptions)} method is called this option will control the time of token expiry,
```

Comment:
Could you please describe the behavior of "<0" and ">=0"? the description about `Admin` can be dropped since that is duplicate to the docs of `expireDelegationToken`

Reply:
I put the description of this behavior inside `@param`, is that ok ?

![image](https://github.com/user-attachments/assets/27ecc2db-ea6a-4031-b81b-6fc5141b9ce7)